### PR TITLE
fix blank page if open configuration page with URL directly

### PR DESCRIPTION
**Pull request by ylwu-amzn**
*Issue #, if available:*

*Description of changes:*

* fix blank page if open configuration page with URL directly

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by ylwu-amzn** [comment]
Local `yarn test:jest` result.

```
Test Suites: 1 skipped, 40 passed, 40 of 41 total
Tests:       4 skipped, 176 passed, 180 total
Snapshots:   33 passed, 33 total
Time:        48.145s
Ran all test suites.
✨  Done in 51.00s.
```

File: public/pages/DetectorConfig/containers/DetectorConfig.tsx [hunk 1]
```diff
@@ -40,20 +40,22 @@ export const DetectorConfig = (props: DetectorConfigProps) => {
...
+            onEditDetector={props.onEditDetector}
+          />
+          <EuiSpacer />
+          <Features
+            detectorId={props.detectorId}
+            detector={detector}
+            onEditFeatures={props.onEditFeatures}
+          />
+        </EuiPageBody>
+      ) : null}
```
Comment (ohltyler) [inline public/pages/DetectorConfig/containers/DetectorConfig.tsx:58]: Do we want to display something here that would give more info to the user, such as if a detector's info is loading or if there was an error retrieving the detector?

Comment (ylwu-amzn) [reply public/pages/DetectorConfig/containers/DetectorConfig.tsx:58]: Good point, will add a loading state

File: public/pages/DetectorConfig/containers/DetectorConfig.tsx [hunk 2]
```diff
@@ -21,14 +21,15 @@ import { RouteComponentProps } from 'react-router';
...
+import { EuiLoadingSpinner } from '@elastic/eui';
 
 interface DetectorConfigProps extends RouteComponentProps {
   detectorId: string;
   onEditFeatures(): void;
   onEditDetector(): void;
 }
 
-export const DetectorConfig = (props: DetectorConfigProps) => {
+export function DetectorConfig(props: DetectorConfigProps) {
```
Comment (yizheliu-amazon) [inline public/pages/DetectorConfig/containers/DetectorConfig.tsx:32]: just curious: why this change is needed?

Comment (ylwu-amzn) [reply public/pages/DetectorConfig/containers/DetectorConfig.tsx:32]: This is to keep consistent with other places.
